services/gtfs-rt-archiver-v3/gtfs_rt_archiver_v3/consumer.py

```diff
@@ -25,12 +25,16 @@ def set_exception_fingerprint(event, hint):
 
     exception = hint["exc_info"][1]
     if isinstance(exception, RTFetchException):
+        # strip out 511 agency ids, since the same outage often affects all of them
+        cleaned_url = re.sub(
```

Comment:
Do we have access here to the original URL from Airtable? Can we just use that for the API key scrubbing?

Reply:
ooh that's a great idea

Reply:
Oh wait no this is separate; the API key issue exists within the body of the nested exception message (i.e. `requests` or `urllib` or something downstream formatting their own exception with the full URL path)